python/tvm/contrib/graph_runtime.py

```diff
@@ -162,7 +162,12 @@ def set_input(self, key=None, value=None, **params):
             keys = list(params.keys())
             keys.sort(key=lambda x: -np.prod(params[x].shape))
             for k in keys:
-                self._get_input(k).copyfrom(params[k])
+                # TODO(zhiics) Skip the weights for submodule in a better way.
+                # We should use MetadataModule for initialization and remove
```

Comment:
Shall we shoot an warning at least if val is not available?

Reply:
Yes, there is a warning in the C++ side.

https://github.com/apache/incubator-tvm/blob/d8c80c382f02052b07da1235190a5b6c7acea994/src/runtime/graph/graph_runtime.cc#L93